use-timescale/troubleshoot-timescaledb.md

```diff
@@ -100,6 +100,46 @@ equal the sum of `timescaledb.max_background_workers` and
 
 For more information, see the [worker configuration docs][worker-config].
 
+### Cannot compress chunk
+
+You might see this error message when trying to compress a chunk if
+the ACL for the compressed hypertable is corrupt:
+
+```sql
+tsdb=> SELECT compress_chunk('_timescaledb_internal._hyper_65_587239_chunk');
+ERROR: role 149910 was concurrently dropped
+```
+
+This can be caused if you dropped a user for the hypertable before
+TimescaleDB 2.5. For this case, the user would be removed from
+`pg_authid` but not revoked from the compressed table. As a result,
+the compressed table will contain an ACL item that refers to a
+numerical value rather than an existing user:
+
+```sql
+tsdb=> \dp _timescaledb_internal._compressed_hypertable_2
```

Comment:
It would be good to explain how to get from the `_timescaledb_internal._hyper_65_587239_chunk` chunk to the `_timescaledb_internal._compressed_hypertable_2` table.

Reply:
Added a paragraph about that.